app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -16,6 +16,11 @@ import org.threeten.bp.LocalDate
 import org.threeten.bp.ZoneOffset
```

Comment:
Please move this change to the test commit.

Reply:
Done.